tests/test_parser.py

```diff
@@ -282,8 +282,9 @@ def test_timestamps_discarded(self):
 b 2  1234567890
 """)
         a = CounterMetricFamily("a", "help", labels=["foo"])
-        a.add_metric(["bar"], 1)
-        b = CounterMetricFamily("b", "help", value=2)
+        a.add_metric(["bar"], 1, timestamp=0)
+        b = CounterMetricFamily("b", "help")
+        b.add_metric([], 2, timestamp=1234567890)
```

Comment:
This isn't right, Prometheus timestamp are in milliseconds whereas the Python client is using the OpenMetrics data model so is working in seconds.